pkgs/top-level/all-packages.nix

```diff
@@ -10553,6 +10553,8 @@ in
 
   gdal_1_11 = callPackage ../development/libraries/gdal/gdal-1_11.nix { };
 
+  gdal_2_4_0 = callPackage ../development/libraries/gdal/2.4.0.nix { };
```

Comment:
gdal_2

Reply:
not also the filename can then be 2.nix. That way it doesn't have to change in case of version bumps. By the way, the latest release is 2.4.2.

Reply:
should I worry about doing a patch release? Honestly i just don't want to rebuild everything that relies on gdal, lol

Reply:
The bump can be done later.